presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -674,6 +701,6 @@ exists as we've rolled back to the previous state.
 Iceberg Connector Limitations
 -----------------------------
 
-* :doc:`/sql/delete` is only supported if the ``WHERE`` clause matches entire partitions.
```

Comment:
I suppose the old text was just wrong?

Reply:
I just fixed this now -- but yes it was wrong